docker-compose.yml

```
@@ -1,23 +1,29 @@
 redis:
   build: ./redis
+  expose:
+    - "6379"
   ports:
     - "6379:6379"
-  net: "host"
   env_file: .env
   container_name: denguetorpedo-redis
   volumes:
-    - <local_folder_for_redis>:/bitnami
+    - /Users/Alan/Documents/UCA-CYT/SE2/2019/denguechat-compose/redis/redisVolume:/bitnami
   restart: always
 
 denguetorpedo:
   build: ./denguetorpedo
+  expose:
+    - "3001"
+    - "5000"
   ports:
     - "3001:3001"
     - "5000:5000"
   command: "./deploy.sh"
-  net: "host"
   container_name: denguetorpedo
   env_file: .env
   volumes:
-  - <local_folder_of_cloned_denguetorpedo_git_project>:/home/dengue/denguetorpedo
-  restart: always
\ No newline at end of file
+    - /Users/Alan/Documents/UCA-CYT/SE2/2019/denguetorpedo:/home/dengue/denguetorpedo
```

Let's avoid local paths
```suggestion
    - <local_folder_of_cloned_denguetorpedo_git_project>:/home/dengue/denguetorpedo
```